src/ecrc-frontend/src/components/page/tou/TOU.js

```diff
@@ -18,12 +30,27 @@ export default function TOU({ page: { header } }) {
     return <Redirect to="/ecrc/bcscRedirect" />;
   }
 
+  const termOfUseOnScroll = event => {
+    const { scrollHeight, scrollTop, clientHeight } = event.target;
+    if (!reachedEnd) {
```

Comment:
why not just do all 3 checks in same if condition?
```
if (!reachedEnd && (scrollHeight - scrollTop <= clientHeight + 5)) {}
```

Reply:
the user could scroll up again. That means in current page, if user has scroll to the bottom once, then we do not care about it again.

Reply:
you have two back to back if statements, you can just combine them the functionality will be the exact same

Reply:
code updated